### PR TITLE
adds focusgroup attribute basic support

### PR DESCRIPTION
**Pull request by mezhnin**
Notice: I had to slightly increase size-limit value in package.json to pass the tests.

### CODE REVIEW

File: focus-group.js [hunk 1]
```diff
@@ -37,12 +40,16 @@ function getToolbarItems(group) {
       item.role === 'button' ||
       item.type === 'button' ||
       item.role === 'checkbox' ||
-      item.type === 'checkbox'
+      item.type === 'checkbox' ||
+      item.hasAttribute('tabindex')
```
Comment (ai) [inline focus-group.js:44]: Let's just ignore `tabindex` since we anyway ignore it in the order

Comment (mezhnin) [reply focus-group.js:44]: If I remove the `tabindex` here, we won't be able to navigate through non-focusable elements (like div or span). Is that ok?

Comment (ai) [reply focus-group.js:44]: Yes. We already added this limitation for other case, so let's keep it since we need a lot of code to support this edge case really (like track order of `tabindex`)

Comment (mezhnin) [reply focus-group.js]: @ai fixed

**Comment by ai** [comment]
We also need:

1. To add to `test/demo`
2. Add section to `README.md` » `Focus Group`. You can just add an example, I will write more text.

**Comment by mezhnin** [comment]
> We also need:
> 
> 1. To add to `test/demo`
> 2. Add section to `README.md` » `Focus Group`. You can just add an example, I will write more text.

There are 3 demo blocks with `focusgroup` attribute in `test/demo` (inline, block and no value). I don't know what else to add.
I will update Readme in a next PR. 
Thanks for the merge.

**Comment by mezhnin** [comment]
> We also need:
> 
> 1. To add to `test/demo`
> 2. Add section to `README.md` » `Focus Group`. You can just add an example, I will write more text.

https://github.com/ai/keyux/pull/47
